variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "function_name" {
+  default = "ssh-notifier"
+}
+
+variable "schedule_expression" {
+  default = "rate(5 minutes)"
+}
+
+# variable "sns_topic_arn" {
+
+# }
+
+variable "elasticsearch_password_parameter_name" {
+  default = "ssh_notifier_elasticsearch_password"
+}
+
+variable "elasticsearch_username" {
+
+}
+
+variable "elasticsearch_hostname" {
+
+}
+
+variable "tag_selector_key" {
+  default = "k8s.io/cluster-autoscaler/node-template/label/PROJECT-SERVICE"
+}
+
+variable "tag_selector_value" {
+
+}
+
+variable "frequency" {
+  default = "5m"
+}
+
+variable "query_string" {
```

Comment:
Can't see a tfvars file for those variables. Are you going to use the tf files as a tf module or do you need to add a tfvars file (secrets excluded of course)?